libraries/classes/Controllers/Table/TableStructureController.php

```diff
@@ -524,7 +524,7 @@ protected function displayHtmlForColumnChange($selected, $action)
         $fields_meta = array();
         for ($i = 0; $i < $selected_cnt; $i++) {
             $value = $this->dbi->getColumns(
-                $this->db, $this->table, $selected[$i], true
+                $this->db, $this->table, addslashes($selected[$i]), true
```

Comment:
```suggestion
                $this->db, $this->table, $this->dbi->escapeString($selected[$i]), true
```

Maybe this would be better ?